website/src/app/blog/using-tauri/readme.mdx

```diff
@@ -0,0 +1,716 @@
+   windows in a subprocess that we can stop and restart when needed
+3. Switch away from Tauri entirely. Iced uses about half as much RAM, and GTK+
+   also has some Windows support and some Rust support.
+
+Hopefully some of that 100 MB is code pages we can share with Edge and other
+WebView2 apps, but I suspect most of it is HTML and JS overhead.
+
+## Ubuntu forwards and backwards compatibility is hard
+
+Ubuntu 20.04 and 24.04 don't have any `webkit2gtk` versions in common.
+
+|              | `webkit2gtk` 4.0 | `webkit2gtk` 4.1 |
+| ------------ | ---------------- | ---------------- |
+| Ubuntu 20.04 | X                |                  |
+| Ubuntu 22.04 | X                | X                |
+| Ubuntu 24.04 |                  | X                |
+
+It's not possible to link with both 4.0 and 4.1, so it's impractical to make one
+exe that runs on all 3 of the Ubuntu versions we'd like to support. This is a
+common problem in Linux GUI programs.
+
+In fact, at time of writing, Tauri 1.x
+[just doesn't work on Ubuntu 24.04](https://github.com/tauri-apps/tauri/issues/9662)
+unless you cheat and add another repo to install the 4.0 package from. Tauri
+2.x, still in beta, won't work on 20.04 since it requires 4.1.
+
+Tauri does offer an AppImage option to bundle the webview, but this increases
+the download size from ~10 MB to ~100 MB, we can't install our connlib service
+if we're running from an AppImage, and AppImages built on 20.04 have rendering
+bugs on 24.04 anyway.
+
+Switching to another GUI framework might work, but since GTK+ is already
+involved in this incompatibility issue, it's likely that GTK+ itself has some
+compatibility problems we'd hit. We'd have to spend time playing with it before
+committing to a port. We also tried building an example Iced app on Ubuntu
+20.04, and it didn't render on 24.04 either. Maybe it's something related to the
+recent Wayland cut-over.
+
+There is a plan that _should_ work, and it doesn't have any big leaps of faith,
+but it's still painful:
+
+- Refactor the code so that we can build for both Tauri 1.x and Tauri 2.x.
+  (Needed sooner or later anyway)
+- Use Docker or something to build one `.deb` inside 20.04 and another inside
+  22.04. (Or worst case, two entire VMs)
+- Unpack the two packages, add a Bash script shim that checks which WebKit is
+  installed and launches the correct binary, then re-pack them into a single
+  `.deb`. (We already re-pack Tauri's `.deb`, so this should be possible)
+
+Or maybe we can install 4.1 on 20.04 somehow. Whatever we do, it won't be
+simple. Worst case, we can give up and use Electron.
+
+## WebView2 just doesn't install sometimes
+
+This is an upstream issue with WebView2 for Windows. Sometimes the installer
+times out in our GitHub CI runners and we don't know why.
+
+<Link href="/images/blog/using-tauri/webview-install-hang.png" target="_blank">
+  <Image
+    width={1230}
+    height={464}
```

Comment:
```suggestion
    height={464}
    className="rounded shadow"
```

Reply:
This will make it consistent with the codeblocks, asciinema view, and other images